src/types.md

```diff
@@ -400,6 +471,33 @@ let bo: Binop = add;
 x = bo(5,7);
 ```
 
+### Other traits
+
+Closure types implement the following traits, if allowed to do so by the
+captured values:
+
+* `[Sized]`
+* `[Send]`
+* `[Sync]`
+* `[Clone]`
+* `[Copy]`
+
+`[Sized]` is always implemented (local variables are all sized, so all captured
```

Comment:
More of a meta-point, but I really don't like having parentheticals like this. They should generally just be another sentence.

Reply:
Removed that sentence anyway.